services/notifications/pkg/config/defaults/defaultconfig.go

```diff
@@ -40,6 +41,13 @@ func DefaultConfig() *config.Config {
 			},
 			RevaGateway: shared.DefaultRevaConfig().Address,
 		},
+		Store: config.Store{
+			Store:    "nats-js-kv",
+			Nodes:    []string{"127.0.0.1:9233"},
+			Database: "notifications",
+			Table:    "",
+			TTL:      192 * time.Hour,
```

Comment:
@bastianbeier after discussing with @wkloucek and @2403905  I agree that 2 weeks are a better TTL. Could you change it back? Sorry, my bad.

Reply:
Done, np :)